src/app/containers/Headings/__snapshots__/index.test.jsx.snap

```diff
@@ -57,6 +57,8 @@ exports[`Headings with subheading data should render correctly 1`] = `
 
 <h2
   className="c0"
+  id="This is a subheading"
```

Comment:
You'll want to strip out spaces and punctuation from the id. Spaces can be replaced by dashes `-`.